.activate-venv.sh

```diff
@@ -17,10 +17,10 @@ fi
 # prefer python2 over python3
 if [ ! -f .use-python-3.flag ]; then
     # highest installed version of py2
-    python=$(which python2)
+    python=$(which python2 python2.7 | head -n 1)
```

Comment:
this is clever. see https://github.com/elifesciences/lax/blob/develop/mkvenv.sh for something similar